contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -251,10 +268,10 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
   event UpkeepPerformed(
     uint256 indexed id,
     bool indexed success,
-    uint32 checkBlockNumber,
+    uint96 totalPayment,
     uint256 gasUsed,
     uint256 gasOverhead,
-    uint96 totalPayment
+    bytes trigger
```

Comment:
arbitrary bytes here cost gas. Is there potential to just to bytes32 hash of the trigger?

Reply:
probably not good for front end though

Reply:
yeah front end needs... will be good for debugging too